src/librustc_back/target/dragonfly_base.rs

```diff
@@ -16,20 +16,10 @@ pub fn opts() -> TargetOptions {
         linker: "cc".to_string(),
         dynamic_linking: true,
         executables: true,
-        linker_is_gnu: true,
```

Comment:
Hm, I wonder why FreeBSD (freebsd_base.rs) removed ```linker_is_gnu``` then. ```ld``` in DragonFly is gnu. So this clearly has to stay.